source/02-FN/01.ptx

```diff
@@ -364,7 +364,7 @@
 </activity>
 
 <remark>
-  Notice in <xref ref="FN1-determine-if-function-from-equation"/> that when you are given an equation with a <m>y^2</m> term, the equation is not a function.
+  Notice that <xref ref="FN1-determine-if-function-from-equation"/> shows that equations with a <m>y^2</m> term generally do not define functions.
```

Comment:
Technically $y^2=0$ defines a function, so this is a little more careful.

Reply:
Also some fun shapes like $y^3=x^2$

Reply:
? That's a function but it doesn't have a $y^2$. And $y^2=x^3$ isn't a function.

Reply:
Students often focus more on the "any higher power of $y$" than the $y^2$ in particular.

Reply:
Point taken - makes me wonder if this remark is overall helpful or has potential to unintentionally mislead...

Reply:
I think if $f$ is any bijective function, and $g$ is just a function, then $f(y)=g(x)$ is a function (as then $y=f^{-1}(g(x))$).  Even if you can't explicitly solve for $f^{-1}$.